<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectInfoDlg</class>
 <widget class="QDialog" name="ProjectInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Edit</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {font: 75 12pt &quot;Arial&quot;;}

#content {
	background-color: rgb(255, 255, 255);
}
/* Customize check boxes. */
QCheckBox {
    spacing: 5px;
}

QCheckBox::indicator {
    width: 13px;
    height: 13px;
}

QCheckBox::indicator:unchecked {
    image: url(:/images/checkbox_unchecked.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/images/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/images/checkbox_unchecked_pressed.png);
}

QCheckBox::indicator:checked {
    image: url(:/images/checkbox_checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/images/checkbox_checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/images/checkbox_checked_pressed.png);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="content" native="true">
     <property name="minimumSize">
      <size>
       <width>650</width>
       <height>320</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>320</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>76</y>
        <width>47</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>249</y>
        <width>411</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Add me as watcher to tickets created by others</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>18</y>
        <width>47</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
     <widget class="QtMaterialAvatar" name="logoWdt" native="true">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>120</width>
        <height>110</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameEdt">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>71</y>
        <width>240</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>47</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Users</string>
      </property>
     </widget>
     <widget class="QtMaterialRaisedButton" name="okBtn">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QtMaterialToggle" name="activeToggle" native="true">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>70</width>
        <height>50</height>
       </rect>
      </property>
     </widget>
     <widget class="QtMaterialToggle" name="watcherToggle" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>70</width>
        <height>50</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="userLstWdt" native="true">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>520</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialToggle</class>
   <extends>QWidget</extends>
   <header>qtmaterialtoggle.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header>qtmaterialraisedbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialAvatar</class>
   <extends>QWidget</extends>
   <header>qtmaterialavatar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
